packages/validation/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@trezor/validation",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "type": "module",
+    "main": "src/index",
+    "scripts": {
+        "lint": "eslint '**/*.{ts,tsx,js}'",
+        "test:unit": "jest",
+        "type-check": "tsc --build"
+    }
+}
```

Comment:
you are missing `@trezor/utils` dependency here

Reply:
done